sycl/doc/extensions/supported/sycl_ext_intel_esimd/examples/README.md

```diff
@@ -65,4 +65,102 @@ is to show the basic ESIMD APIs in well known examples.
           });
     });
     ```
-3) TODO: Add more examples here.
+3) Dot Product Accumulate Systolic (DPAS) API - ["dpas"](./dpas.md)
+
+   Please see the full source code here: ["dpas"](./dpas.md)
+
+   ```c++
+    // Res = A * B.
+    // Assume the HW is PVC.
+
+    constexpr int SystolicDepth = 8;
```

Comment:
nit: i think we are currently doing constexpr vars or defines in `CAPS_AND_UNDERSCORES` code style, but maybe that's only for globals constexprs? not a big deal either way

Reply:
It looks like it is mostly `CapitalizedWords` in esimd headers.